README.md

```diff
@@ -12,3 +12,57 @@ The _RFC - Py-Spiffe features and use cases_ document is public at: https://docs
 * Gomez Coronel Andres (HPE)
 * Lambrecht Max (HPE)
 
+
+## Development setup
+
+### Prerequisites
+For basic development you will need:
+
+* Python 3.6
+* Pipenv (https://github.com/pypa/pipenv)
+* Pyenv (https://github.com/pyenv/pyenv)
+
+### Setup the environment
+
+Use `pyenv` to install Python 3.6. At the moment of writing latest 3.6 version is 3.6.12
+```
+pyenv install 3.6.12
+```
+
+Clone the repository
+```
+git clone git@github.com:HewlettPackard/py-spiffe.git
+```
+
+cd to the py-spiffe directory
+```
+cd py-spiffe
+```
+
+Set the Python version using the version installed via pyenv
+```
+pyenv local 3.6.12
+```
+
+Create the virtual environment
+```
+make env
+```
+`.venv` directory is created using `pipenv` on the root directory of the repo.
+
+
+Then install all the `dev` dependencies
+```
+make dev
+```
+
+Run the tests
+```
+make test
+```
+
+To remove the virtual environment
```

Comment:
I would put this step in separate section `Clean up your dev environment`

Reply:
I liked that. Thanks Max